odiglet/go.mod

```diff
@@ -1,6 +1,8 @@
 module github.com/odigos-io/odigos/odiglet
 
-go 1.22.0
+go 1.22.5
```

Comment:
@RonFed why these changed ?

Reply:
I think its because I ran `go mod tidy` locally, I'll try to revert this part and see if the CI is happy this it